contentcuration/contentcuration/utils/automation_manager.py

```diff
@@ -0,0 +1,51 @@
+from contentcuration.utils.recommendations import Recommendations
+from contentcuration.utils.recommendations import RecommendationsAdapter
+
+
+class AutomationManager:
+    def __init__(self):
+        self.reccomendations_backend_instance = Recommendations()
```

Comment:
I think this should be the `[RecommendationsBackendFactory](https://github.com/akolson/studio/blob/7a197291a7a3d51f38c363a6aa1f85638f4e6a90/contentcuration/contentcuration/utils/recommendations.py#L38)`